src/test/java/edu/hm/hafner/util/StringContainsUtilsTest.java

```diff
@@ -31,4 +32,15 @@ void shouldSearchStrings() {
         assertThat(containsAnyIgnoreCase("This is a string text.", "wrong", "is")).isTrue();
         assertThat(containsAnyIgnoreCase("This is a string text.", "wrong", "IS")).isTrue();
     }
+
+    @Test
+    void shouldSearchAllStrings() {
+        assertThat(containsAllIgnoreCase("This is a string text.", "something")).isFalse();
```

Comment:
Um die Methode containsAllIgnoreCase zu testen, wären auch Tests sinnvoll die das Handling von null-Referenzen testen. Zum Beispiel so wie in der Methode shouldHandleNull in dieser Klasse.